processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -332,6 +313,39 @@ public void cleanup(CloseableGrouperIterator<RowBasedKey, Row> iterFromMake)
     );
   }
 
+  private List<ReferenceCountingResourceHolder<ByteBuffer>> getMergeBuffersHolder(
+      int numBuffers,
+      boolean hasTimeout,
+      long timeoutAt
+  )
+  {
+    try {
+      if (numBuffers > mergeBufferPool.maxSize()) {
+        throw new ResourceLimitExceededException(
```

Comment:
This error should include something like "Try raising druid.processing.numMergeBuffers."

Reply:
Added.